sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -14,10 +14,6 @@
 
 - Renamed `Azure::Core::Amqp::Models::AmqpMessageFormatValue` to `AmqpDefaultMessageFormatValue`.
```

Comment:
@LarryOsterman Is this AMQP changelog complete? I thought there were more changes.

Reply:
@ahsonkhan, Larry's PR just went into main, and I just updated this PR merging main to this PR, and his changes are here (expand the section above - the "Rationalized..." and "additional AMQP error values" Features are here, as well as the "changed return values" Breaking Change).